src/driver/odoc_driver.ml

```diff
@@ -705,13 +734,19 @@ module Dune_mode = struct
 end
 
 module Opam_mode = struct
-  let run packages packages_dir = run (Opam { packages; packages_dir })
+  let run libs packages packages_dir =
+    run (Opam { libs; packages; packages_dir })
 
   let packages =
     (* TODO: Is it package or library? *)
-    let doc = "The packages to document" in
+    let doc = "The packages to document (mutually exclusive with -l)" in
     Arg.(value & opt_all string [] & info [ "p" ] ~doc)
 
+  let libs =
+    (* TODO: Is it package or library? *)
+    let doc = "The libraries to document (mutually exclusive with -p)" in
+    Arg.(value & opt_all string [] & info [ "l" ] ~doc)
+
```

Comment:
Should these 2 be subcommands of odoc_driver opam instead? this code accepts empty lists for both but does it make sense?